mycli/main.py

```diff
@@ -1159,7 +1159,13 @@ def cli(database, user, host, port, socket, password, dbname,
         mycli.run_cli()
     else:
         stdin = click.get_text_stream('stdin')
-        stdin_text = stdin.read()
+        try:
+            stdin_text = stdin.read()
+        except MemoryError:
+            click.secho('Failed! Ran out of memory.' ,err=True, fg='red')
+            click.secho('You might want to try the official mysql client.' ,err=True, fg='red')
+            click.secho('Sorry... :(' ,err=True, fg='red')
```

Comment:
Shouldn't the commas have no space preceding them but a single space afterwards too?

Reply:
Looks like the lint checker actually complained about that: https://travis-ci.org/dbcli/mycli/jobs/456439367. And yet it exited with 0. We need to fix that.

Reply:
Fixed the pep8 violation.